tests/ha/filesystem.pm

```diff
@@ -86,20 +86,20 @@ sub run {
     if (is_node(1)) {
         # Create the Filesystem resource
         assert_script_run
-"EDITOR=\"sed -ie '\$ a primitive $fs_rsc ocf:heartbeat:Filesystem params device='$fs_lun' directory='/srv/$fs_rsc' fstype='$fs_type''\" crm configure edit";
+"EDITOR=\"sed -ie '\$ a primitive $fs_rsc ocf:heartbeat:Filesystem params device='$fs_lun' directory='/srv/$fs_rsc' fstype='$fs_type''\" crm configure edit", $default_timeout;
 
         # Only OCFS2 can be cloned
         if ($fs_type eq 'ocfs2') {
-            assert_script_run "EDITOR=\"sed -ie 's/^\\(group base-group.*\\)/\\1 $fs_rsc/'\" crm configure edit";
+            assert_script_run "EDITOR=\"sed -ie 's/^\\(group base-group.*\\)/\\1 $fs_rsc/'\" crm configure edit", $default_timeout;
         }
         else {
             if ($resource eq 'drbd_passive') {
-                assert_script_run "EDITOR=\"sed -ie '\$ a colocation colocation_$fs_rsc inf: $fs_rsc ms_$resource:Master'\" crm configure edit";
-                assert_script_run "EDITOR=\"sed -ie '\$ a order order_$fs_rsc Mandatory: ms_$resource:promote $fs_rsc:start'\" crm configure edit";
+                assert_script_run "EDITOR=\"sed -ie '\$ a colocation colocation_$fs_rsc inf: $fs_rsc ms_$resource:Master'\" crm configure edit", $default_timeout;
```

Comment:
Strangely Travis didn't complain about this, maybe it will with the "fix"?

Reply:
Yes, neither travis nor `make test` complained about this, but you're right. I'll fix the indentation. Also travis is failing due to the second declaration of `post_fail_hook` in `tests/ha/setup_hosts_and_luns`, so need to fix that as well. It didn't fail with `make test`

Reply:
So, with this fix `make test` complains, so I think travis will as well. Will fix only the `post_fail_hook` problem to check whether it's necessary or not.

Reply:
Travis is passing, so I would leave this as is. With `make test` that indentation fails, so I'm assuming that travis would fail as well.

Reply:
> Travis is passing, so I would leave this as is. With `make test` that indentation fails, so I'm assuming that travis would fail as well.

Ok, it's good for me.